Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:latest AS CLIENT
+
+RUN git clone https://github.com/geoffrey45/swing-client.git client
+
```

Comment:
I think we should git clone this repo from the latest tag (for now). The code might have changed since the latest release.

Reply:
I just made a commit that should use the latest tag

Reply:
Thanks.